app/packages/core/src/plugins/SchemaIO/components/ImageView.tsx

```diff
@@ -16,6 +45,8 @@ export default function ImageView(props) {
         height={height}
         width={width}
         alt={alt}
+        onClick={onClick}
+        style={{ cursor: href ? "pointer" : "default" }} // Change cursor based on href
```

Comment:
We might want to make this a setting as well. Its weird to not get the `pointer` when the intention is that the image is clickable. Maybe something like:

```python
panel.img('img', ..., pointer_on_hover=True)
```